spec/integration/fetching_content_item_spec.rb

```diff
@@ -1,217 +1,59 @@
-        end
-      end
-
-      it "should set a cache-control header with value public" do
-        get "/content/vat-rates"
-        expect(response.headers["Cache-Control"]).to eq('public')
-      end
-
-      describe "adjusting expiry for publish intents" do
-        it "should set the Expires header to the date of the upcoming publish_intent" do
-          Timecop.freeze do
-            create(:publish_intent, :base_path => "/vat-rates", :publish_time => 23.minutes.from_now)
-            get "/content/vat-rates"
-            expect(response.headers["Expires"]).to eq(23.minutes.from_now.httpdate)
-          end
-        end
-
-        it "should set the Expires header to 30 mins with publish_intent more than 30 mins away" do
-          Timecop.freeze do
-            create(:publish_intent, :base_path => "/vat-rates", :publish_time => 40.minutes.from_now)
-            get "/content/vat-rates"
-            expect(response.headers["Expires"]).to eq(30.minutes.from_now.httpdate)
-          end
-        end
-
-        it "should set the Expires header to 30 mins with a publish_intent in the past" do
-          Timecop.freeze do
-            create(:publish_intent, :base_path => "/vat-rates", :publish_time => 10.minutes.ago)
-            get "/content/vat-rates"
-            expect(response.headers["Expires"]).to eq(30.minutes.from_now.httpdate)
-          end
-        end
-
-        it "should set an Expires header to 5 seconds with a publish_intent that's very recently in the past" do
-          Timecop.freeze do
-            create(:publish_intent, :base_path => "/vat-rates", :publish_time => 10.seconds.ago)
-            get "/content/vat-rates"
-            expect(response.headers["Expires"]).to eq(5.seconds.from_now.httpdate)
-          end
-        end
-      end
+    it "sets cache headers to expire in the default TTL" do
+      expect(cache_control["max-age"]).to eq(default_ttl.to_s)
     end
 
-    describe "expanding linked items" do
-      # functional behaviour of link expansion covered in end_to_end_spec
-
-      describe "generating the API URL" do
-        let!(:linked_item) { create(:content_item, :with_content_id) }
-        before :each do
-          item.links["related"] = [linked_item.content_id]
-          item.save!
-        end
-
-        it "includes the API URL for linked items" do
-          get "/content#{item.base_path}"
-
-          expect(response.status).to eq(200)
-
-          data = JSON.parse(response.body)
-          expect(data["links"]["related"].first["api_url"]).to eq("http://www.example.com/content#{linked_item.base_path}")
-        end
-
-        it "includes the public API URL when requested through the public endpoint" do
-          get "/api/content#{item.base_path}"
-
-          expect(response.status).to eq(200)
-
-          data = JSON.parse(response.body)
-          expect(data["links"]["related"].first["api_url"]).to eq("http://www.example.com/api/content#{linked_item.base_path}")
-        end
```

Comment:
I don't see these 2 tests covered in the restructured tests.  These specifically cover the detail where the `api_url` is different depending which route you request the item through (ie with and without the `/api` prefix)